File: kyu_7/001_Vowel_Count/javascript/js/main.js
// solution 1
function getCount(str) {
  let result = 0;

  for (let i = 0; i < str.length; i++) {
    if ("aeiou".includes(str[i])) result++;
  }

  return result;
}

console.log(getCount("abracadabra")); // 5

// solution 2
function getCount(str) {
  let result = 0;
  let match = "aeiou";

  for (let i = 0; i < str.length; i++) {
    for (let j = 0; j < match.length; j++) {
      if (str[i] === match[j]) {
        result++;
        continue;
      }
    }
  }

  return result;
}

console.log(getCount("abracadabra")); // 5

File: kyu_7/001_Vowel_Count/c/main.c
#include <stddef.h>

size_t get_count(const char *s)
{
	int i, j;
	char *match = "aeiou";
	size_t result = 0;

	for (i = 0; s[i]; i++)
	{
		for (j = 0; match[j]; j++)
		{
			if (s[i] == match[j])
			{
				result++;
				break;
			}
		}
	}

    return result;
}
